about/logo/index.md

```diff
@@ -9,17 +9,18 @@ image:
   creditlink:
 ---
 
-The Stan name and logo are registered trademarks of
-[NumFOCUS](http://numfocus.org/) under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).
+The Stan name and logo are registered trademarks of [NumFOCUS](http://numfocus.org/)
+under the direction of the [Stan Governing Body](/about#stan-governing-body-sgb).
 
-Both may be freely used when referencing Stan, such as in a blog post or lecture
-notes, but they may not be used in the branding of commercial entities, such
-as software packages or paid courses, without the express license of the Stan
-leadership board.  If you have any questions or would like to inquire about
-licensing the Stan name and/or logo for a commercial venture then please contact
-NumFOCUS.
+Both may be freely used when referencing Stan, such as in blog posts, lecture notes,
+or opensourced software packages, but they may not be used in the branding of commercial entities,
+such as paid software packages or paid courses, without the express license of the Stan leadership board.
+If you have any questions or would like to inquire about licensing the Stan name
+and/or logo for a commercial venture then please contact NumFOCUS.
 
-When using the Stan logo please use the official versions linked below,
+Under in no circumstances our name and logo cannot be used in ways that appear (to a casual observer) official, affiliated, or endorsed by Stan, unless you have written permission from the Stan Governing Body.
```

Comment:
This needs to be written---it's not grammatical English and uses a confusing double negation.  The "no circumstances" is redundant.  Also, it's "may" when talking about permission vs. "can" when talking about ability.  Stan is software, so I think you mean the Stan project here.

So I'd recommend changing this to:

```
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```

Reply:
```suggestion
The Stan name and Stan logo may not be used in ways that suggest the usage is endorsed by the Stan project without written permission from the Stan Governing Body.
```